<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabResults</class>
 <widget class="QWidget" name="TabResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <widget class="QStackedWidget" name="resultsStackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>99</horstretch>
        <verstretch>99</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0" columnstretch="0">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="99,99">
         <item row="0" column="0">
          <widget class="QTreeView" name="proxyViewH">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>99</horstretch>
             <verstretch>99</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTextEdit" name="lineDisplayH">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>99</horstretch>
             <verstretch>99</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>99</horstretch>
        <verstretch>99</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTreeView" name="proxyViewV"/>
         </item>
         <item row="1" column="0">
          <widget class="QTextEdit" name="lineDisplayV"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="layoutPushButton">
     <property name="text">
      <string>&amp;Horizontal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
